src/thunderbots/software/CMakeLists.txt

```diff
@@ -174,7 +174,6 @@ if (CATKIN_ENABLE_TESTING)
             backend_output/grsim/grsim_backend.cpp
             backend_output/grsim/grsim_backend.h
             backend_output/backend.h
-            backend_output/grsim/motion_controller.cpp
             ai/primitive/move_primitive.cpp
```

Comment:
I still don't think `move_primitive` and everything else included in this test below `backend_output/backend.h` needs to be here? We shouldn't need the extra stuff since the `grsim_backend_tests` weren't changed.

Reply:
I removed the unnecessary files. Motion controller file is needed though as sendPrimtives uses it now